apps/plea/views.py

```diff
@@ -92,11 +92,11 @@ def save(self, *args, **kwargs):
 
         return super(PleaOnlineForms, self).save(*args, **kwargs)
 
-    def render(self):
+    def render(self, request, request_context=dict):
```

Comment:
Sorted, thanks for helping clear out some bad knowledge I picked up somewhere along the line.